theurgy.lic

```diff
@@ -62,6 +62,66 @@ def clean_altar
   put_in_container($settings.water_holder)
 end
 
+def prayer_bead
+  return if DRSkill.getrank('Mechanical Lore') < 140
+
+  fput "stow #{GameObj.right_hand.noun}" if GameObj.right_hand.name != 'Empty'
+  fput "stow #{GameObj.left_hand.noun}" if GameObj.left_hand.name != 'Empty'
+  get_from_container('prayer chain')
```

Comment:
I had hoped this would be resolved because of the check against commune truff that will be in your YAML - that will only function if you have a prayer chain (and will be noted as such) so this is really just being added to theurgy so you can keep your prayer chain full with the added benefit of a large devotion boost.  I haven't added that check yet because I haven't finished commune truff for combat-trainer yet - but essentially if you don't use that in training-abilities theurgy.lic will skip right past this.